### ISSUE TITLE
Help Wanted - /search pager - stuck?

### ISSUE THREAD

**Issue by Nicole-Ridgwell-NMMNHS** [ArctosDB/arctos#9417]
Help us understand your request (check below):

- [ ] search or data request (help with SQL or provide the specific fields you are trying to retrieve for your collection)
- [ ] add a new Github user to the Arctos Users team
- [ ] a new barcode series
- [ ] a new term for data entry or management 
- [ ] a change or new report or label
- [ ] update to an Agent (split, merge, or other type of change)
- [ ] bulkload changes (to Agents, Identifiers, etc that are not already available as a bulkloading tool)
- [ ] new feature or otherwise a good idea to make Arctos even better
- [ ] quarantine taxon name and suggested replacement (please supply links)
- [ ] API key request (see https://handbook.arctosdb.org/documentation/api.html for requirements)
- [x ] error or bug (please include enough specific information to reproduce)
- [ ] general inquiry (or email arctoscontact@googlegroups.com)
- [ ] access issue (see https://arctosdb.org/faq/)
- [ ] other

**Describe what you're trying to do** 

When I am logged into Arctos on a non-operator account through my google login, I can search results and select to see 1000 results per page, but it will still only show me 100 results AND won't give me the option to go to the next page. If I select 100 results, it works just fine and I can go to the next page of results.

------------

Please review the [Arctos Code of Conduct](https://arctosdb.org/code-of-conduct/) to ensure respectful communication, and see <https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html> for more information about submitting and managing Issues.

**Comment by dustymc** [comment]
I cannot replicate, I see...

<img width="214" height="180" alt="Image" src="https://github.com/user-attachments/assets/fd7633b6-dd11-47ce-a5c0-3a49da707802" />

... which suggests your browser is caching this in some way?

**Comment by mkoo** [comment]
Very different for me-- non-operator account on Chrome:

<img width="602" height="382" alt="Image" src="https://github.com/user-attachments/assets/c266235f-5666-4841-9797-e0ddf7fad74a" />

I can see page2 and download. What are your local details? (FYI This browser instance has very few extensions-- just screengrabber and password manager)

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Huh, it is working now.

Edit, nevermind, it shows I can go next page now, but there are not actually 1000 results per page. It says showing 1 to 1,000, but only actually shows 100.

Using Chrome, no extensions.

I'm not too worried about this for myself as I rarely use the non-operator account, but am concerned about public users running into this.

With the operator account I only have the option to view 100 at a time.

**Comment by mkoo** [comment]
Nicole this may possibly be due to how many customised results you are displaying? 

Also I just tested this in Chrome: Google account lets me view 1000, however using my Github account, then I can only see 100 at a time. 

I am not sure why you reopened this issue but this seems to be a browser display issue which we can document as a possibly consideration if someone chooses to log in with one account over another. Let me know if there's a specific action you're requesting but otherwise I think we should close

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
No longer getting the option to view 1000 records which fixes the issue.
